README.md

```diff
@@ -276,6 +276,22 @@ If key generation didn't work or you want to use a custom key and certificate, y
 pyrdp-mitm.py 192.168.1.10 -k private_key.pem -c certificate.pem
 ```
 
+##### Monster-in-the-Middle NLA/CredSSP connections
+You can also specify the server's key and certificate along with the
```

Comment:
TODO
- [ ] link to blogpost introducing the feature
- [ ] link to cert extraction documentation in `docs/`